docs/source/asr/intro.rst

```diff
@@ -54,6 +54,5 @@ The full documentation tree is as follows:
    api
    resources
    examples/kinyarwanda_asr.rst
-   examples/esperanto_asr/esperanto_asr.rst
```

Comment:
We should move out kiraynwana as well (can be another PR)

Reply:
Yep let's do a separate PR